bedrock/base/views.py

```diff
@@ -171,6 +171,11 @@ def server_error_view(request, template_name='500.html'):
     return l10n_utils.render(request, template_name, status=500)
 
 
+def page_not_found_view(request, template_name='404.html'):
```

Comment:
I think Paul commented on this in Slack, ybut you may need `exception` in here as an argument.
```suggestion
def page_not_found_view(request, exception, template_name='404.html'):
```